lib/tapjoy/ldap/key/remove.rb

```diff
@@ -0,0 +1,92 @@
+module Tapjoy
+  module LDAP
+    module Key
+      # Remove a user key from user profile
+      class Remove
+        # Remove key from LDAP
+        def remove
+          keys  # Get keys first
+          Tapjoy::LDAP::Key.verify_user(opts[:user], results)
+
+          confirm unless opts[:force]
+          Tapjoy::LDAP::client.replace_attribute(
+            @user_dn, :sshPublicKey, keep_keys)
```

Comment:
@user_dn won't be set here until current_keys has been invoked by keep_keys, which hasn't happened yet.